integration/test/chartvalues/chartvalues_test.go

```diff
@@ -47,4 +73,28 @@ func TestChartValues(t *testing.T) {
 	if err != nil {
 		t.Fatalf("could not install %q %v", cr, err)
 	}
+	err = r.WaitForStatus(cr, "DEPLOYED")
+	if err != nil {
+		t.Fatalf("expected %#v got %#v", nil, err)
+	}
+	l.Log("level", "debug", "message", fmt.Sprintf("%s succesfully deployed", cr))
+
+	// Check if chart got created by chart-operator
+	err = r.WaitForStatus(testChartRelease, "DEPLOYED")
+	if err != nil {
+		t.Fatalf("expected %#v got %#v", nil, err)
+	}
+	l.Log("level", "debug", "message", fmt.Sprintf("%s succesfully deployed", testChartRelease))
+
+	// Check if values are applied
+	rc, err := helmClient.GetReleaseContent(testChartRelease)
+	if err != nil {
+		t.Fatalf("could not get release content of %s %v", testChartRelease, err)
+	}
+	l.Log("level", "debug", "message", fmt.Sprintf("chart %s has values %#v", testChartRelease, rc.Values))
+
+	expectedValue := "values-config"
+	if rc.Values["config"] != expectedValue {
```

Comment:
This is the actual test if chart-operator correctly applied the provided values (in the configmap) to the helm chart.